package.json

```diff
@@ -6,7 +6,7 @@
     "url": "git@github.com:braintree/framebus"
   },
   "homepage": "https://github.com/braintree/framebus",
-  "version": "5.1.2",
+  "version": "6.0.0",
```

Comment:
Let's hold off on doing this until we're actually ready to publish it, that way we can use the `npm version major` command to automatically bump it and the packate-lock.json and tag it in git.

```suggestion
  "version": "5.1.2",
```

Reply:
got it, I'll revert the change.